.gitpod.yml

```diff
@@ -105,6 +105,7 @@ ports:
 
 vscode:
   extensions:
+    # cSpell:disable
```

Comment:
?

Reply:
Extensions have mostly usernames, not really worth spell checking IMO

Reply:
No you have two comments here, should ir not be one?

Reply:
Ah the second comment enables cSpell for any lines that might be added below

Reply:
Diff looked straange